src/components/comment-input.component.js

```diff
@@ -113,12 +120,12 @@ export class CommentInput extends Component {
                   : translate('issue.main.commentInput', language)
               }
               multiline
-              blurOnSubmit
+              blurOnSubmit={false}
               onChangeText={text => this.setState({ text })}
               onContentSizeChange={event =>
                 this.setState({ height: event.nativeEvent.contentSize.height })}
               onSubmitEditing={event =>
```

Comment:
I just tried to post a comment with new line from the iOS simulator and nothing was posted... <img width=300 alt="simulator screen shot 12 aout 2017 a 21 28 22" src="https://user-images.githubusercontent.com/9287184/29243686-6389f3ca-7fa5-11e7-9921-e74337773060.png">